config/default/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-      - image: quay.io/openshift/cluster-api-provider-baremetal:master
+      - image: quay.io/openshift/cluster-api-provider-baremetal:main
```

Comment:
I wonder if this is actually a real thing (we don't push directly to quay, do we?)